templates/email/subscription-activated.hypr

```diff
@@ -0,0 +1,23 @@
+{% extends "email/email" %}
+{% block body-content %}
+
+<!--- Header --->
+<div class="mz-curbside-email-header">  
+<img src="https://kibocommerce.com/wp-content/themes/kibo/images/logo.png" alt="logo">
+<div class="mz-curbside-header-body">
+    <p><strong>{{ siteContext.generalSettings.websiteName }}</strong></p>
+    <h3>{{ labels.curbsideOrderConfirmationtxt }}</h3>
+</div>
+</div>
```

Comment:
Do we need this Header section with Kibo logo and website name? I don't see it in the mock. Confirm with Aalekh once. If yes then we'll have to add it to other templates as well.

Reply:
removed the header part